src/components/CodeOverview/index.spec.tsx

```diff
@@ -276,6 +276,33 @@ describe(__filename, () => {
     expect(link).toHaveProp('title', `Jump to line ${line}`);
   });
 
+  it('renders links for a code line using a custom getCodeLineAnchor', () => {
+    const customAnchor = 'example-anchor';
+    const fakeGetCodeLineAnchor = jest.fn(() => customAnchor);
+
+    const location = createFakeLocation();
+    const root = renderWithFixedHeight({
+      getCodeLineAnchor: fakeGetCodeLineAnchor,
+      content: generateFileLines({ count: 3 }).join('\n'),
+      location,
+    });
+
+    const line = 3;
+    const lineIndex = line - 1;
+    const link = root.find(Link).at(lineIndex);
+
+    expect(link).toHaveProp(
+      'to',
+      expect.objectContaining({
+        hash: customAnchor,
+      }),
+    );
+
+    expect(fakeGetCodeLineAnchor).toHaveBeenCalled();
+    expect(fakeGetCodeLineAnchor.mock.calls[lineIndex]).toEqual([line]);
+    expect(fakeGetCodeLineAnchor.mock.calls[lineIndex - 1]).toEqual([line - 1]);
```

Comment:
Are you looking for https://jestjs.io/docs/en/expect.html#tohavebeennthcalledwithnthcall-arg1-arg2- or https://jestjs.io/docs/en/expect.html#tohavebeenlastcalledwitharg1-arg2-? We should try to not direct access `mock.calls`.

Reply:
Cool, I didn't know that existed. Thanks.